src/custom/state/claim/hooks/index.ts

```diff
@@ -22,17 +22,27 @@ export * from './hooksMod'
 // TODO: replace with real repo when known
 export const CLAIMS_REPO = 'https://raw.githubusercontent.com/gnosis/cow-mrkl-drop-data-chunks/final/chunks/'
 
-export enum ClaimType {
-  Airdrop, // free, no vesting, can be available on both mainnet and gchain
-  GnoOption, // paid, with vesting, must use GNO, can be available on both mainnet and gchain
-  UserOption, // paid, with vesting, must use Native currency, can be available on both mainnet and gchain
-  Investor, // paid, with vesting, must use USDC, only on mainnet
-  Team, // free, with vesting, only on mainnet
-  Advisor, // free, with vesting, only on mainnet
+export const enum ClaimType {
```

Comment:
@alfetopito 's code i believe this was a numeric enum, 0,1,2,3,4 etc, is this intended to be string enum now?

Reply:
We discussed it today over the meeting but just for posterity sake, it should be a number enum as it was